src/proto/FunctionRpc.proto

```diff
@@ -255,6 +255,18 @@ message InvocationRequest {
 
   // binding metadata from trigger
   map<string, TypedData> trigger_metadata = 4;
+
+  // Populates activityId, tracestate and tags from host
+  TraceContext trace_context = 5;
+}
+
+// Host sends ActivityId, traceStateString and Tags from host
+message TraceContext {
+	string Traceparent = 1;
```

Comment:
add comments on each of the properties what is maps to on host. is Traceparent = ActivityId?